src/scripts/browserDeviceProfile.js

```diff
@@ -247,9 +247,13 @@ function supportedDolbyVisionProfilesHevc(videoTestElement) {
             .replace(/no/, '')) {
             supportedProfiles.push(5);
         }
-        if (videoTestElement
-            .canPlayType('video/mp4; codecs="dvh1.08.09"')
-            .replace(/no/, '')) {
+        if (
+            videoTestElement
+                .canPlayType('video/mp4; codecs="dvh1.08.09"')
+                .replace(/no/, '')
+            // LG TVs from at least 2020 onwards should support profile 8, but they don't report it.
+            || (browser.web0s && browser.web0sVersion > 5)
```

Comment:
Testing for `web0sVersion` is enough (if not webOS, it is undefined and the check will be false).
```suggestion
            || browser.web0sVersion > 5
```

So webOS 5 doesn't support DoVi, but its updates (5.1, 5.2; _I don't know the exact versions) do, right?